internal/v2/command/org.go

```diff
@@ -132,7 +132,7 @@ func (r *CommandSide) addOrg(ctx context.Context, organisation *domain.Org) (_ *
 	orgAgg := OrgAggregateFromWriteModel(&addedOrg.WriteModel)
 	orgAgg.PushEvents(org.NewOrgAddedEvent(ctx, organisation.Name))
 	for _, orgDomain := range organisation.Domains {
-		if err := r.addOrgDomain(ctx, orgAgg, NewOrgDomainWriteModel(orgAgg.ID(), orgDomain.Domain), orgDomain); err != nil {
+		if _, err := r.addOrgDomain(ctx, orgAgg, NewOrgDomainWriteModel(orgAgg.ID(), orgDomain.Domain), orgDomain, claimedUserIDs...); err != nil {
```

Comment:
as long as claimedUserIDs can be passed, there's the possibility addOrgDomain will return userAggregates, so these must be handled